server/src/sse.rs

```diff
@@ -0,0 +1,63 @@
+use std::sync::{Arc, Mutex};
+use actix_web_lab::sse;
+use actix_web_lab::sse::Event;
```

Comment:
Please fix those unused imports. Changes look fine otherwise, I’ll approve once this is fixed

Reply:
But the build fails without those 3?

Reply:
Code formatted and removed unused imports